Formula/orocos-kdl.rb

```diff
@@ -0,0 +1,33 @@
+class OrocosKdl < Formula
+  desc "Orocos Kinematics and Dynamics C++ library"
+  homepage "http://www.orocos.org/kdl"
+  url "https://github.com/orocos/orocos_kinematics_dynamics/archive/v1.3.1.tar.gz"
+  sha256 "aff361d2b4e2c6d30ae959308a124022eeef5dc5bea2ce779900f9b36b0537bd"
+
+  depends_on "cmake" => :build
+  depends_on "eigen@3.2"
+
+  def install
+    cd "orocos_kdl" do
+      system "cmake", ".", "-DEIGEN3_INCLUDE_DIR=#{Formula["eigen@3.2"].opt_include}/eigen3",
+                           *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <kdl/frames.hpp>
+      int main()
+      {
+        using namespace KDL;
+        Vector v1(1.,0.,1.);;
+        Vector v2(1.,0.,1.);
+        assert(v1==v2);
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "-Wall", "test.cpp", "-L#{lib}", "-lorocos-kdl", "-o", "test"
```

Comment:
does this work while unlinked

Reply:
Will test

Reply:
I unlinked the `eigen` dependency and `brew test orocos-kdl` worked.
```
$ brew unlink eigen 
Unlinking /usr/local/Cellar/eigen/3.3.4... 4 symlinks removed
$ brew linkage orocos-kdl
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
Dependencies with no linkage:
  eigen
$ brew test orocos-kdl
Testing orocos-kdl
==> /usr/bin/clang++ -Wall test.cpp -L/usr/local/Cellar/orocos-kdl/1.3.1/lib -lorocos-kdl -o test
==> ./test
```
Is that what you wanted?

Reply:
also orocos-kdl itself

Reply:
Nope.  I get the following.
```
$ brew test orocos-kdl 
Error: orocos-kdl is not linked
```
How do I fix something like that?

Reply:
`-f`

Reply:
I think `brew` prevents testing unlinked formla.  From `/usr/local/Homebrew/Library/Homebrew/dev-cmd/test.rb` 

```
      # Don't test unlinked formulae
      if !ARGV.force? && !f.keg_only? && !f.linked?
        ofail "#{f.full_name} is not linked"
        next
      end
```

Reply:
correct. `-f`.

Reply:
Well that should have been obvious ;)

Fixed.